### DISCUSSION TITLE
[Unraid] Authelia doesn't start automatically

### DISCUSSION THREAD

**Discussion by domodial** [authelia/authelia#3554] [Q&A]
Hello,

I have a problem with authelia.
It works perfectly well, the only thing is that it does not start automatically with Unraid.

The button is however in the on position, like the other containers.

Is there anything specific to do ?

**Comment by james-d-elliott** [comment]
Authelia runs like any other container or binary, there is nothing specific to Authelia that would prevent automatic startup.

**Reply by domodial** [reply]
this is precisely what I don't understand, out of the 25 containers, only Authelia does not start at boot. The container autostart option is enabled. Maybe this Unraid option does not work with Authelia and should a specific parameter be passed ?

**Reply by james-d-elliott** [reply]
> Maybe this Unraid option does not work with Authelia and should a specific parameter be passed ?

Unlikely. It runs on docker as you expect (though in docker it's called a restart policy). Unraid uses docker. No other user (there are 100s) using unraid has reported this. 

Make sure you've read the unraid docs on this: https://wiki.unraid.net/Manual/Docker_Management#Controlling_container_auto-start

**Comment by Borlean** [comment]
I had this happen on Unraid after auto updates of containers and believe it has to do with redis/maria_db not being fully started when Authelia starts. I'm trying to add the --restart unless-stopped extra argument to the Authelia container to see if that works.
